src/sentry/static/sentry/app/views/settings/organizationProjects/index.jsx

```diff
@@ -38,13 +39,15 @@ class OrganizationProjects extends AsyncView {
 
   getEndpoints() {
     const {orgId} = this.props.params;
+    const itemsPerPage = 50;
     return [
       [
         'projectList',
         `/organizations/${orgId}/projects/`,
         {
           query: {
             query: this.props.location.query.query,
+            per_page: itemsPerPage,
```

Comment:
How are subsequent pages loaded?

Reply:
Hmm yeah I think we need this? https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/asyncComponent.tsx#L245

Reply:
upon further exploration with Billy it seems like the pagination component should take care of adding the query parameters for multiple pages. I also tested locally with ITEMS_PER_PAGE=2 and it functions as expected.

Reply:
TIL about the auto created `${stateKey}PageLinks` properties.